python/sparkdl/graph/builder.py

```diff
@@ -67,17 +69,101 @@ def __exit__(self, *args):
     def run(self, *args, **kargs):
         """
         This method delegate the TensorFlow graph execution
-        to the underlying tf.Session object to perform 
+        to the underlying tf.Session object to perform
         one step of graph computation.
 
         All the parameters are defined according to `tf.Session.run`
-        Reference: https://www.tensorflow.org/api_docs/python/tf/Session#run        
+        Reference: https://www.tensorflow.org/api_docs/python/tf/Session#run
         """
         return self.sess.run(*args, **kargs)
 
+    def asUDF(self, udf_name, fetches, feeds_to_fields_map=None, blocked=False, register=True):
+        """
+        Register the graph in this session as a Spark SQL UserDefinedFunction.
+
+        The following example creates a UDF that takes the input
+        from a DataFrame column named 'image_col' and produce some random prediction.
+
```

Comment:
Our code examples require setting up spark and tensorflow/keras for testing. Maybe we can provide them as examples and import them in the doc?